src/plugins/filters/components/FiltersView.vue

```diff
@@ -90,14 +90,20 @@ export default {
         this.composition.load();
         this.unobserve = this.openmct.objects.observe(this.providedObject, 'configuration.filters', this.updatePersistedFilters);
         this.unobserveGlobalFilters = this.openmct.objects.observe(this.providedObject, 'configuration.globalFilters', this.updateGlobalFilters);
-        this.unobserveAllMutation = this.openmct.objects.observe(this.providedObject, '*', (mutatedObject) => this.providedObject = mutatedObject);
+        if (this.providedObject.isMutable !== true) {
+            this.unobserveAllMutation = this.openmct.objects.observe(this.providedObject, '*', (mutatedObject) => {
+                this.providedObject = mutatedObject;
+            });
+        }
     },
     beforeDestroy() {
         this.composition.off('add', this.addChildren);
         this.composition.off('remove', this.removeChildren);
         this.unobserve();
         this.unobserveGlobalFilters();
-        this.unobserveAllMutation();
+        if (this.unobserveAllMutation) {
```

Comment:
We should be able to remove the * observer altogether

Reply:
Done